### PR TITLE
#162163148 Users should be able to report an article

### PR DESCRIPTION
**Pull request by alexkayabula**
### What does this PR do?
This Pull Request enables users to report articles that violate Authors Haven terms and agreements.

### Description of Task to be completed?
When a user visits the Authors Haven application, he/she should be able to report any articles they find to violate Authors Haven terms and agreements. 

### How should this be manually tested?
- Clone the repository then checkout to the branch `ft-report-article-162163148 `.
- Install the required dependencies.
     - yarn 
- Start the server.
     - yarn start
- Run the project in the browser.
- Access the report article option to report any article.
screen shot 2019-01-31 at 05 53 53
screen shot 2019-01-31 at 05 28 18
screen shot 2019-01-31 at 05 28 52
screen shot 2019-01-31 at 05 29 44

#### What are the relevant pivotal tracker stories?
[#162163148](https://www.pivotaltracker.com/story/show/162163148)

### CODE REVIEW

**Review by ongebo — APPROVED** [review]
Looks good to me.

**Review by sekayasin — APPROVED** [review]
LGTM

**Review by mulondo — APPROVED** [review]

File: src/actions/ArticlesActions.js [hunk 1]
```diff
@@ -159,4 +162,34 @@ export const dislikeArticle = (slug, payload) => dispatch => {
...
+    headers: {
+      'content-type': 'application/json',
+      authorization: `Bearer ${token}`
+    },
+    CORS: 'no-cors',
+    body: JSON.stringify({ reason: `${payload.reason}` })
+  })
+    .then(res => res.json())
+    .then(response => {
+      console.log(response)
```
Comment (RrNn) [inline src/actions/ArticlesActions.js]: Remove this `console.log`

File: src/actions/socialAction.js [hunk 2]
```diff
@@ -18,14 +18,19 @@ export const responseGoogle = response => dispatch => {
         data.user.auth_token ===
           'A user with this email already exists, Please login'
       ) {
+        console.log(dispatch);
```
Comment (RrNn) [inline src/actions/socialAction.js]: Remove this `console.log`

File: src/actions/socialAction.js [hunk 3]
```diff
@@ -18,14 +18,19 @@ export const responseGoogle = response => dispatch => {
...
         dispatch({
           type: GET_ERRORS,
           payload: data
         });
         alert(data.user.auth_token);
       } else {
-        localStorage.setItem('Token', data.user.auth_token);
-        window.location = '/';
+        localStorage.setItem('token', data.user.auth_token);
+        console.log(data.user.username);
```
Comment (RrNn) [inline src/actions/socialAction.js]: Remove these 2 `console.logs`

**Review by RrNn — CHANGES_REQUESTED** [review]

File: src/actions/ArticlesActions.js (hunk 1 above)
Comment (alexkayabula) [reply src/actions/ArticlesActions.js]: It has been removed.

File: src/actions/socialAction.js (hunk 3 above)
Comment (alexkayabula) [reply src/actions/socialAction.js]: They have  been removed

**Pull request merged**